libr/util/sys.c

```diff
@@ -26,6 +26,7 @@
 #include <r_lib.h>
 
 static char** env = NULL;
+static char** origenv = NULL;
```

Comment:
I don't particularly like this global `origenv`... even though there was already a global `env` as well :(

Reply:
Me neither but i don't see any other option here